sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -187,6 +189,65 @@ def hpat_pandas_stringmethods_{methodname}_impl(self{methodparams}):
 """
 
 
+@overload_method(StringMethodsType, 'find')
+def hpat_pandas_stringmethods_find(self, sub, start=0, end=None):
+    """
+    Pandas Series method :meth:`pandas.core.strings.StringMethods.find()` implementation.
+
+    Note: Unicode type of list elements are supported only. Numpy.NaN is not supported as elements.
+
+    .. only:: developer
+
+    Test: python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_find
+
+    Parameters
+    ----------
+    self: :class:`pandas.core.strings.StringMethods`
+        input arg
+    sub: :obj:`str`
+        Substring being searched
+    start: :obj:`int`
+        Left edge index
+        *unsupported*
+    end: :obj:`int`
+        Right edge index
+        *unsupported*
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
+    """
+
+    ty_checker = TypeChecker('Method find().')
+    ty_checker.check(self, StringMethodsType)
+
+    if not isinstance(sub, (StringLiteral, UnicodeType)):
+        ty_checker.raise_exc(sub, 'str', 'sub')
+
+    accepted_types = (Integer, IntegerLiteral, NoneType, Omitted)
+    if not isinstance(start, accepted_types) and start != 0:
+        ty_checker.raise_exc(start, 'None, int', 'start')
+
+    if not isinstance(end, accepted_types) and end is not None:
+        ty_checker.raise_exc(end, 'None, int', 'end')
+
+    def hpat_pandas_stringmethods_find_impl(self, sub, start=0, end=None):
+        if start != 0:
```

Comment:
Why we are not supporting `start` and `end`? UnicodeString doesn't?

Reply:
Numba 0.46 doesn't support str.find with extra parameters.